package/MDAnalysis/coordinates/PDB.py

```diff
@@ -275,16 +275,23 @@ def __init__(self, filename, **kwargs):
         pdbfile = self._pdbfile = util.anyopen(filename)
 
         line = "magical"
+        windows_overbyte = 0
         while line:
             # need to use readline so tell gives end of line
             # (rather than end of current chunk)
             line = pdbfile.readline()
+            # the tell() text opaque number striding is not portable
+            # to windows so compensate accordingly
+            if os.name == 'nt':
+                # for a text file mode these aren't necessarily
+                # bytes; var name for simplicity only
+                windows_overbyte += 1
```

Comment:
So if I do run the following code under windows and python I will get different results?
```python
pos = f.tell()
f.seek(0)
f.seek(pos)
print(f.tell())
```

Reply:
As [indicated on stackoverflow](https://stackoverflow.com/a/19498291/2942522) `file.tell()` can return illegal values on Windows when parsing a text file. The parsing really should be happening in 'rb' mode as the link to the Python standard lib docs indicates, if we want to keep using that kind of approach with tell().

In fact, our test file has Unix-style line endings, which is the exact pathological scenario described by the [standard lib docs](https://docs.python.org/2/library/stdtypes.html#file.tell).

Reply:
thanks for the clarification. Can we change the read mode to `rb` then?

Reply:
Probably -- I'm not sure if we can use anyopen in binary mode; could just circumvent and open the file object manually though, maybe.

Then there's the question of mixing text parsing with binary input; strikes me as potentially a bit of an endeavor to work with the heterogeneity of possible PDB files, though could be more robust once finalized.

Reply:
we need `anyopen` for zip files. That is a good argument that binary mode isn't such a good choice. It would good to have links to the docs than in the comment and a sentence why binary mode might not be an easy solution for us.